pyext/src/rmsd_calculation.py

```diff
@@ -325,31 +351,35 @@ def get_rmfs_coordinates_one_rmf(path, rmf_A, rmf_B, subunit_name=None,
                             residue_in_bead + "_" + str(copy_number))
             mod_id += 1
 
+    if symm_groups_file:
+        for grp in symm_groups:
+            if len(grp)==0:
+                print("Warning. Symmetry option specified but created \
+                        symmetry group is empty. Cross-check the \
+                        specification of symmetry groups.")
```

Comment:
This will look super-ugly on output. You probably want something like this instead:

    print("Warning. Symmetry option specified but created "
          "symmetry group is empty. Cross-check the "
          "specification of symmetry groups.")